mne/preprocessing/ieeg/_projection.py

```diff
@@ -0,0 +1,90 @@
+# Authors: Alex Rockhill <aprockhill@mailbox.org>
+#
+# License: BSD-3-Clause
+
+from os import path as op
+
+from itertools import combinations
+import numpy as np
+
+from ...io.pick import _picks_to_idx
+from ...surface import _read_mri_surface
+from ...transforms import (apply_trans, invert_transform, _cart_to_sph,
+                           _ensure_trans)
+from ...utils import verbose, get_subjects_dir, _validate_type
+
+
+@verbose
+def project_sensors_onto_brain(info, trans, subject, subjects_dir=None,
+                               picks=None, n_neighbors=10, verbose=None):
+    """Project sensors onto the brain surface.
+
+    .. note:: This is useful in ECoG analysis for compensating for
+              "brain shift" or shrinking of the brain away from the
+              skull due to changes in pressure during the craniotomy.
+
+    .. note:: To use the brain surface, a BEM model must be created
+              e.g. using :ref:`mne watershed_bem` using the T1 or
+              :ref:`mne flash_bem` using a FLASH scan.
+
+    Parameters
+    ----------
+    %(info_not_none)s Note, ``info`` is modified in place.
```

Comment:
we usually do not modify parameters in place with plain functions. At least if we do this we should have an inplace parameter so this behavior can be discovered more easily. I have seen many new users coming from matlab trip on this issue. If you can above this it would be great. thx @alexrockhill